### PR TITLE
PR to merge artifacts for release 0.3

### PR DESCRIPTION
**Pull request by jmaron99**
@gilbode , @markxnelson - Please review and approve this PR required for the 0.3 release. Build is running at https://build.verrazzano.io/job/verrazzano/job/release-merge-0-3/

### CODE REVIEW

**Review by gilbode — APPROVED** [review]

**Review by gilbode — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Reporting Security Vulnerabilities
+
+Oracle values the independent security research community and believes that responsible disclosure of security vulnerabilities helps us ensure the security and privacy of all our users.
+
+Please do NOT raise a GitHub Issue to report a security vulnerability. If you believe you have found a security vulnerability, please submit a report to secalert_us@oracle.com preferably with a proof of concept. We provide additional information on [how to report security vulnerabilities to Oracle](https://www.oracle.com/corporate/security-practices/assurance/vulnerability/reporting.html) which includes public encryption keys for secure email.
+
+We ask that you do not use other channels or contact project contributors directly. 
+
+Non-vulnerability related security issues such as new great new ideas for security features are welcome on GitHub Issues.
```
Comment (rosemarymarano) [inline SECURITY.md:9]: issues such as new great new ideas for security features -> issues such as, great new ideas for security features,

**Review by rosemarymarano — COMMENTED** [review]
Security.md could use an edit. I stopped reviewing it after I saw that this PR was merged.

File: SECURITY.md (hunk 1 above)
Comment (jmaron99) [reply SECURITY.md:9]: If there are doc issues we'd like to address, let's track them with a doc JIRA or the equivalent and target the next release

Comment (rosemarymarano) [reply SECURITY.md:9]: ok, I'll make that happen. Thank you.

**Pull request merged**